electron/index.ts

```diff
@@ -25,6 +28,57 @@ import * as errors from '../www/model/errors';
 import {ConnectionStore, SerializableConnection} from './connection_store';
 import * as process_manager from './process_manager';
 
+// Imports the server list from Outline Beta, if one exists, by copying localstorage from Outline
```

Comment:
Is it worth moving this code to a separate file?

Reply:
Mmm, good idea. Moved to `beta_migration.ts`.